src/firefly/js/ui/NaifidPanel.jsx

```diff
@@ -182,18 +161,23 @@ function renderSuggestion(str){
 
 function handleValueChange(payload, fireValueChange) {
     const newPayload= {...payload};
-    if (!payload.valid) {
-        newPayload.value= '';
-        newPayload.feedback= '';
+    newPayload.valid  = Boolean(newPayload.value) || (newPayload.displayValue === '' || newPayload.displayValue === naifNamevalue);
+    if (!newPayload.valid && !newPayload.message) {
+        if (!newPayload.message) {
```

Comment:
should be 
```
    if (!newPayload.valid) {
        if (!newPayload.message) {
```

Reply:
Done thanks!

Reply:
Done thanks!

Reply:
For suggestions list sorting, this will be in a new ticket.